packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -203,6 +220,40 @@ const BlockInspector = ( { showNoBlockSelectedMessage = true } ) => {
 		);
 	}
 
+	if (
+		isOffCanvasNavigationEditorEnabled &&
+		blockInspectorAnimationSettings
+	) {
+		const animationOrigin =
+			blockInspectorAnimationSettings &&
+			blockInspectorAnimationSettings.enterDirection === 'leftToRight'
+				? -50
+				: 50;
+
+		return (
+			<motion.div
+				animate={ {
+					x: 0,
+					opacity: 1,
+					transition: {
+						ease: 'easeInOut',
+						duration: 0.14,
+					},
+				} }
+				initial={ {
+					x: animationOrigin,
+					opacity: 0,
+				} }
+				key={ selectedBlockClientId }
+			>
+				<BlockInspectorSingleBlock
+					clientId={ selectedBlockClientId }
+					blockName={ blockType.name }
+				/>
```

Comment:
This block of code is duplicated as the default `return` statement.

Could we consider conditionally wrapping the default `return` in the motion div and avoid the duplication?